orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/MonitoredDeployStrategy.groovy

```diff
@@ -89,13 +75,15 @@ class MonitoredDeployStrategy implements Strategy {
     def savedCapacity = stage.context.savedCapacity ?: stage.context.capacity?.clone()
     stage.context.savedCapacity = savedCapacity
 
-    // FIXME: this clobbers the input capacity value (if any). Should find a better way to request a new asg of size 0
     stage.context.capacity = [
       min    : 0,
       max    : 0,
       desired: 0
     ]
 
+    // Don't allow old-school "rollback" key in the deploy stage, we handle our own rollback
+    stage.context.remove("rollback")
```

Comment:
In which cases will we have "rollback" here as one of the stages?

Reply:
this is a context entry that is used by other strategies (and added by the UI if you use, say RedBlack)
I am removing it to reduce confusion when looking at execution JSON  since Monitored will not honor this setting - it will use `failureActions` instead. So it should never really be here, unless the user had a deploy stage using RB and then switched to Monitored strategy on that deploy stage